flink-streaming-java/src/main/java/org/apache/flink/streaming/util/typeutils/FieldAccessor.java

```diff
@@ -157,15 +156,15 @@ public T set(T record, F fieldValue) {
 
 		SimpleTupleFieldAccessor(int pos, TypeInformation<T> typeInfo) {
```

Comment:
accessing fields in a `Row` will fail because `Row` does not extend `Tuple`. For a proper fix, we would need a `RowFieldAccessor` and use that one when we deal with a `DataStream<Row>`. We would then need to add the `RowFieldAccessor` to the `FieldAccessorFactory`.